insights/parsers/openvswitch_other_config.py

```diff
@@ -0,0 +1,41 @@
+"""
+OpenvSwitchOtherConfig - command ``ovs-vsctl -t 5 get Open_vSwitch . other_config``
+===================================================================================
+
+Class OpenvSwitchOtherConfig process the output of the following OpenvSwitch command:
+
+ovs-vsctl -t 5 get Open_vSwitch . other_config
+
+
+Sample input::
+
+    {dpdk-init="true", dpdk-lcore-mask="30000003000", dpdk-socket-mem="4096,4096", pmd-cpu-mask="30000003000"}
+
+Examples:
+
+    >>> ovs_other_conf = shared[OpenvSwitchOtherConfig]
```

Comment:
Since shared does not exist anymore in 3.x, better change it in examples to something more consistent with 3.x. Please refer to tomcat_xml.py or jboss_versoin.py